.azure-pipelines/scripts/ut/env_setup.sh

```diff
@@ -99,7 +99,7 @@ elif [[ $(echo "${test_case}" | grep -c "tf pruning") != 0 ]]; then
 fi
 
 if [[ $(echo "${test_case}" | grep -c "api") != 0 ]] || [[ $(echo "${test_case}" | grep -c "adaptor") != 0 ]]; then
-    pip install git+https://github.com/intel/auto-round.git@a868c805de4be271cfe7403309a64d9bf03a0ecf
+    pip install git+https://github.com/intel/auto-round.git@de05114b27bcab3dd24d0a7d9b57447c9dfbe91e
```

Comment:
Please update the commit: b65830f3f6cb32d92a5c8ba5f80ace12d517357b

Reply:
updated https://github.com/intel/neural-compressor/pull/1641/commits/ea834e775eb836c4705b84b34c3e293e314f75cc